lib/index.js

```diff
@@ -41,13 +41,12 @@ const prependFood = (food) => {
 
 const deleteFood = (event) => {
   event.preventDefault();
-  var foodId = $('.delete-food-btn').data('food-id');
-  return fetch(`https://evening-peak-25213.herokuapp.com/api/v1/foods` + '/' + foodId, {
+  var foodId = event.currentTarget.dataset.foodId
```

Comment:
this works as well- I think we have been using Jquery to grab values of variables in existing code: 
var foodId = $('.delete-food-btn').data('food-id');
  return fetch(`https://evening-peak-25213.herokuapp.com/api/v1/foods` + '/' + foodId

Reply:
I got the delete to autorefresh per your comments! Great solution, thanks for that! I'll check into this now

Reply:
I got this resolved as well! Thanks for checking this out. Ready to be merged when you are!